aws-iam-policies/docs/restricted-policy-doc-1.json5

```diff
@@ -293,6 +293,23 @@
         // Upgrade needs old/new instance status
       ],
       "Resource": "*"
+    },
+    {
+            "Sid": "StartStopRDS",
+            "Effect": "Allow",
+            "Action": [
+                "rds:StartDBInstance",
+                // Stop RDS Instance while stopping the cluster
+                "rds:StopDBInstance",
+                // Start RDS Instance while starting the cluster
+                "rds:DescribeDBInstances"
```

Comment:
Might have to remove from [here](https://github.com/cloudera/cdw-cloud-policies/blob/main/aws-iam-policies/docs/restricted-policy-doc-2.json5#L133), since it is the same policy just that we split it into 2 as AWS character limit on each policy file.